src/ripple/rpc/handlers/LedgerRequest.cpp

```diff
@@ -118,28 +119,27 @@ Json::Value doLedgerRequest (RPC::Context& context)
         ledgerHash = neededHash ? *neededHash : zero; // kludge
     }
 
-    auto ledger = ledgerMaster.getLedgerByHash (ledgerHash);
+    // Try to get the desired ledger
+    auto ledger = context.app.getInboundLedgers().acquire (
+        ledgerHash, ledgerIndex, InboundLedger::fcGENERIC);
+
+    if (context.app.config().standalone())
```

Comment:
Even in standalone mode we might have the ledger in the node database, so we do want to call `acquire` in that case. I think the right fix is to change the `if` in line 127 to `! ledger && ...`